examples/gno.land/r/gov/dao/voter.gno

```diff
@@ -0,0 +1,57 @@
+package govdao
+
+import (
+	"std"
+)
+
+const YAY = "YES"
+const NAY = "NO"
+
+func NewPercentageVoter(percent int) *PercentageVoter {
+	if percent < 0 || percent > 100 {
```

Comment:
Shouldn't it be `<= 0`?

Reply:
If you want to enable "no votes" executions then the original condition is good

Reply:
Yep, I wanted to support 0, so no votes are needed